MUXSDKStats/MUXSDKStats/MUXSDKPlayerBinding.m

```diff
@@ -250,6 +276,7 @@ - (void)detachAVPlayer {
         [_timeUpdateTimer invalidate];
         _timeUpdateTimer = nil;
     }
+    [_masterPlaylistRequest cancel];
```

Comment:
It is important we cancel any requests for the master manifest that have not yet completed. In the case of change video, we don't want the first request completing after the new video has started,